os/ProtectedLibs.mk

```diff
@@ -153,7 +153,7 @@ TINYARALIBS += $(LIBRARIES_DIR)$(DELIM)lib_wps$(LIBEXT)
 endif
 
 ifeq ($(CONFIG_AMEBALITE_WIFI),y)
-TINYARALIBS += $(LIBRARIES_DIR)$(DELIM)lib_enc$(LIBEXT)
+# TINYARALIBS += $(LIBRARIES_DIR)$(DELIM)lib_enc$(LIBEXT)
```

Comment:
@gexuyan @fengyunlai @ziyik Could you review and confirm this change?

Reply:
Dear @sunghan-chang , This change is ok as it just follows the FlatLibs.mk file, and I have confirmed with Thapa. Thank you.